src/App.tsx

```diff
@@ -1,25 +1,47 @@
-import React from 'react';
-import logo from './logo.svg';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
 import './App.css';
 
+export interface Breed{
```

Comment:
```suggestion
type Breed = {
```
Não estamos usando OOP quando adotamos a abordagem funcional do React então é preferivel user type.

Reply:
c09dbff09dec73953c50c62548e64a95a5f9c387